config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [OPTIONAL] A more rails way of configuring these routes would be:

   ```ruby
   resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
   end
   ```
   
   Please consider this going forward.

Reply:
Duly noted

Thanks a lot for this insight...

🙇